erdpy/delegation/staking_provider.py

```diff
@@ -122,10 +122,10 @@ def prepare_args_modify_delegation_cap(args: Any):
 def prepare_args_automatic_activation(args: Any):
     data = 'setAutomaticActivation'
     if args.set:
-        data += '@' + binascii.hexlify(str.encode('yes')).decode()
+        data += '@' + binascii.hexlify(str.encode('true')).decode()
```

Comment:
`hexlify` is not needed here at all. Please do the following instead:
```suggestion
        data += '@' + 'true'.encode('ascii').hex()
```